model/src/application/command/option.rs

```diff
@@ -71,84 +71,86 @@ impl<'de> Deserialize<'de> for CommandOption {
     }
 }
 
+// "Required options must be listed before optional options"
+// <https://discord.com/developers/docs/interactions/application-commands#application-command-object-application-command-option-structure>
```

Comment:
this comment refers to the list of options in the `options` field, not the list of fields in this object.